go/vt/vtgate/evalengine/testcases/cases.go

```diff
@@ -1315,6 +1316,56 @@ var JSONExtract_Schema = []*querypb.Field{
 	},
 }
 
+func FnElt(yield Query) {
+	for _, s1 := range inputStrings {
+		for _, n := range inputBitwise {
+			yield(fmt.Sprintf("ELT(%s, %s)", n, s1), nil)
+		}
+	}
+
+	for _, s1 := range inputStrings {
+		for _, s2 := range inputStrings {
+			for _, n := range inputBitwise {
+				yield(fmt.Sprintf("ELT(%s, %s, %s)", n, s1, s2), nil)
+			}
+		}
+	}
+
+	for _, s1 := range inputStrings {
+		for _, s2 := range inputStrings {
+			for _, s3 := range inputStrings {
+				for _, n := range inputBitwise {
+					yield(fmt.Sprintf("ELT(%s, %s, %s, %s)", n, s1, s2, s3), nil)
+				}
+			}
+		}
+	}
+
+	validIndex := []string{
+		"1",
+		"2",
+		"3",
+	}
+	for _, s1 := range inputStrings {
+		for _, s2 := range inputStrings {
+			for _, s3 := range inputStrings {
+				for _, n := range validIndex {
+					yield(fmt.Sprintf("ELT(%s, %s, %s, %s)", n, s1, s2, s3), nil)
+				}
+			}
+		}
+	}
```

Comment:
One thing to look at with stuff like this, is how much test runtime we're adding. It might be too much with all these permutations? How long does it take to run these tests?

Reply:
about 24 sec, in the comparison test

Reply:
I think we should reduce this then, that's a lot of time for a single subtest here.

Reply:
Removed the test for 4 inputs. Total test time reduced to 6 sec, total `56234` number of tests.